nav2_bt_navigator/src/bt_navigator.cpp

```diff
@@ -16,6 +16,7 @@
 #include <memory>
 #include <fstream>
 #include <streambuf>
+#include <set>
```

Comment:
Looks like you're not using `set` but including it to remove the linter error. Use `\\ NOLINT' instead.